crates/rspack_plugin_javascript/src/parser_plugin/import_meta_plugin.rs

```diff
@@ -262,3 +262,30 @@ impl JavascriptParserPlugin for ImportMetaPlugin {
     None
   }
 }
+
+// use when parser.import_meta is false
+pub struct ImportMetaDisabledPlugin;
+
+impl JavascriptParserPlugin for ImportMetaDisabledPlugin {
+  fn member(
```

Comment:
should check `for_name` here? Seems this will replace all member expressions

Reply:
Fixed!

Added conditional check for meta property of ImportMeta kind.

Reply:
Update:

Moved to use `meta_property` hooks instead in case of direct call to `import.meta` - just to align with webpack.

Reply:
@LingyuCoder Please check again